### ISSUE TITLE
Run hadolint container on an AArch64 (ARMv8) doesn't work

### ISSUE THREAD

**Issue by kevinimbrechts** [hadolint/hadolint#314]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

Hello,
I want to run Hadoline on an Alpine distrib (for Raspberri Pi 3) and, when I execute the command from the readme, I have this error message :
```
$ docker run --rm -i hadolint/hadolint < Dockerfile
exec user process caused "exec format error"
```

### Expected behavior
The command must validate a Dockerfile.

### Actual behavior
An error appears when I run this command on Alpine 3.9.2 (aarch64).

### Steps to reproduce the behavior
Just run the hadolint container on Alpine 3.9.2 (aarch64).

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**
Same error :

```bash
standard_init_linux.go:207: exec user process caused "exec format error"
```

**Additional environment details (OS, stack version, etc.)**

- Alpine 3.9.2 (aarch64)
- Docker version 18.09.1-ce, build 4c52b901c6cb019f7552cd93055f9688c6538be4

**Comment by lorenzo** [comment]
We don’t offer an alpine based docker image, can you please explain what you are doing?

**Comment by kevinimbrechts** [comment]
I'm not talking about an Alpine based Docker Image. I have installed an Alpine OS (not a container) on a Raspberry Pi 3 and I have installed Docker on it.
Now, I would like use Hadolint with you Docker command. That's all

**Comment by zemanlx** [comment]
@kevinimbrechts Thanks for raising your issue. 

Alpine is not the problem as it is running in our Docker image. `hadolint` binary is compiled for `x86_64` (amd64) architecture and Rapsberry Pi 3 is built on `aarch64` (ARMv8). I have googled a bit and I do not think that compiling Haskell for ARM is anywhere near production ready state, therefore I am afraid that we are not able to support `aarch64` architecture right now. 

Anyway if you find out how to compile it using CI and `stack` and would be more than happy to include `aarch64` binaries here.

**Comment by kevinimbrechts** [comment]
Thanks @zemanlx. Yes, I have come to the same conclusion. I have tried to build your image and the build crashes when we install GHC:
`RUN stack --no-terminal --install-ghc test --only-dependencies`

**Comment by kevinimbrechts** [comment]
It seems like we can manualy download Haskell Stack for AArch64 architecture : https://docs.haskellstack.org/en/stable/install_and_upgrade/#manual-download_2

What do you think ?

**Comment by lorenzo** [comment]
The first challenge is finding an AArch machine we can use for testing. Is there any CI service that offers this?

**Comment by kevinimbrechts** [comment]
Good question @lorenzo. I don't know.

**Comment by kevinimbrechts** [comment]
This seems work for GHC :
```
apt-get install --no-install-recommends -y libnuma-dev=2.0.11-2.1

WORKDIR /tmp
RUN curl -sSL https://s3.amazonaws.com/public.west.spy.net/ghc/ghc-8.6.3-aarch64-unknown-linux.tar.xz -o ghc-8.6.3-aarch64-unknown-linux.tar.xz && \
    tar xf ghc-8.6.3-aarch64-unknown-linux.tar.xz && \
    cd ghc-8.6.3 && \
    ./configure && \
    make install
```

Now, I have to install Stack.

I'll keep you informed.

**Comment by lorenzo** [comment]
closing in favor of #411
